lib/classes/Error.js

```diff
@@ -1,28 +1,49 @@
 'use strict';
+const chalk = require('chalk');
 
-const ServerlessError = class ServerlessError extends Error {
-  constructor(message, messageId) {
+module.exports.SError = class ServerlessError extends Error {
+  constructor(message) {
     super(message);
     this.name = this.constructor.name;
     this.message = message;
-    this.messageId = messageId;
-    Error.captureStackTrace(this, this.constructor.name);
+    Error.captureStackTrace(this, this.constructor);
   }
 };
 
-module.exports = ServerlessError;
-
-module.exports.errorCodes = {
-  UNKNOWN: 1,
-  MISSING_HOMEDIR: 2,
-  MISSING_AWS_CREDS_PROFILE: 3,
-  MISSING_AWS_CREDS: 4,
-  INVALID_PROJ_NAME: 5,
-  ZIP_TOO_BIG: 6,
-  INVALID_PROJECT_SERVERLESS: 7,
-  NO_LAMBDAS_TAGGED_DEPLOYABLE: 8,
-  ACCESS_DENIED: 9,
-  ENV_KEY_NOT_SET: 10,
-  INVALID_RESOURCE_NAME: 11,
-  NOT_IN_SERVERLESS_PROJECT: 12,
+module.exports.logError = (e) => {
+  const consoleLog = (message) => {
+    console.log(message); // eslint-disable-line no-console
+  };
+
+  const errorType = e.name.replace(/([A-Z])/g, ' $1');
+  let line = '';
+  while (line.length < 56 - errorType.length) {
+    line = `${line}-`;
+  }
+
+  consoleLog(chalk.yellow(` ${errorType} ${line}`));
+
+  if (e.name !== 'ServerlessError') consoleLog(' ');
+
+  consoleLog(chalk.yellow(`     ${e.message}`));
```

Comment:
We should add one newline above and one below it so that we can simply throw an error and the format looks always the same.

<img width="431" alt="bildschirmfoto 2016-06-23 um 09 14 00" src="https://cloud.githubusercontent.com/assets/1606004/16294761/e1721274-3922-11e6-8dea-4e3f1f55a718.png">
<img width="429" alt="bildschirmfoto 2016-06-23 um 09 14 07" src="https://cloud.githubusercontent.com/assets/1606004/16294762/e172a3e2-3922-11e6-830c-ebd7dc87adb8.png">